src/Conference/Plugin.php

```diff
@@ -0,0 +1,322 @@
+<?php
+/**
+ * The main Conference Schedule plugin service provider: it bootstraps the plugin code.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Conference
+ */
+
+namespace TEC\Conference;
+
+use Tribe__Autoloader;
+use Tribe__Main;
+
+/**
+ * Class Plugin
+ *
+ * @since   TBD
+ *
+ * @package TEC\Conference
+ */
+class Plugin {
+	/**
+	 * Stores the version for the plugin.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public const VERSION  = '1.0.0';
```

Comment:
Use the Plugin Register version here.
```suggestion
	public const VERSION  = Plugin_Register::VERSION;
```

Reply:
I originally had it that way, but because common is not loaded when this class loads it throws a fatal error because Tribe__Abstract_Plugin_Register is not available yet until common loads.

Reply:
I reversed it and use the Plugin::VERSION in the Plugin Register